`test_properties` is failing

```
============================= test session starts ==============================
platform linux -- Python 3.12.8, pytest-8.3.3, pluggy-1.5.0
rootdir: /build/source
configfile: pyproject.toml
collected 57 items                                                             

tests/test_language.py ...........F.                                     [ 22%]
tests/test_lookahead_iterator.py .                                       [ 24%]
tests/test_node.py ...............                                       [ 50%]
tests/test_parser.py ..............                                      [ 75%]
tests/test_query.py ...........                                          [ 94%]
tests/test_tree.py ...                                                   [100%]

=================================== FAILURES ===================================
_________________________ TestLanguage.test_properties _________________________

self = <tests.test_language.TestLanguage testMethod=test_properties>

    def test_properties(self):
        lang = Language(self.python)
        self.assertEqual(lang.version, 14)
        self.assertEqual(lang.node_kind_count, 275)
>       self.assertEqual(lang.parse_state_count, 2795)
E       AssertionError: 2809 != 2795
```

Observed on NixOS.

--- a/tests/test_language.py
+++ b/tests/test_language.py
@@ -25,7 +25,7 @@ class TestLanguage(TestCase):
         lang = Language(self.python)
         self.assertEqual(lang.version, 14)
         self.assertEqual(lang.node_kind_count, 275)
-        self.assertEqual(lang.parse_state_count, 2795)
+        self.assertEqual(lang.parse_state_count, 2809)
         self.assertEqual(lang.field_count, 32)
 
     def test_node_kind_for_id(self):

